inference-engine/samples/hello_nv12_input_classification/main.cpp

```diff
@@ -89,14 +151,57 @@ void setBatchSize(CNNNetwork& network, size_t batch) {
     network.reshape(inputShapes);
 }
 
+std::vector<Blob::Ptr> readInputBlobs(std::vector<UString>& data, size_t width, size_t height) {
+    // read image with size converted to NV12 data size: height(NV12) = 3 / 2 * logical height
+
+    // Create tensor descriptors for Y and UV blobs
+    const InferenceEngine::TensorDesc y_plane_desc(InferenceEngine::Precision::U8, {1, 1, height, width},
+                                                   InferenceEngine::Layout::NHWC);
+    const InferenceEngine::TensorDesc uv_plane_desc(InferenceEngine::Precision::U8, {1, 2, height / 2, width / 2},
+                                                    InferenceEngine::Layout::NHWC);
+    const size_t offset = width * height;
+
+    std::vector<Blob::Ptr> blobs;
+    for (auto& buf : data) {
+        // --------------------------- Create a blob to hold the NV12 input data -------------------------------
+        auto ptr = &buf[0];
+
+        // Create blob for Y plane from raw data
+        Blob::Ptr y_blob = make_shared_blob<uint8_t>(y_plane_desc, ptr);
+        // Create blob for UV plane from raw data
+        Blob::Ptr uv_blob = make_shared_blob<uint8_t>(uv_plane_desc, ptr + offset);
+        // Create NV12Blob from Y and UV blobs
+        blobs.emplace_back(make_shared_blob<NV12Blob>(y_blob, uv_blob));
+    }
+
+    return blobs;
+}
+
+bool isBatchedBlobSupported(const Core& ie, const std::string& device_name) {
+    const std::vector<std::string> supported_metrics =
```

Comment:
Nitpick: if data structure is supposed to be used as search-in, I'd use `std::set`/`std::unordered_set`. It's more reasonable if  duplicates are not allowed.

Reply:
Unfortunately, such metrics returned from plugins have the type `std::vector` conversion `std::vector` to a _set_ is not effective  in this case.